cloud-init.txt

```diff
@@ -40,11 +40,12 @@ runcmd:
         apt install aziot-identity-service=1.2.0~rc4-1
         apt install aziot-edge=1.2.0~rc4-1 
 
-        mkdir /etc/aziot
-        wget https://raw.githubusercontent.com/Azure/iotedge-vm-deploy/1.2.0-rc4/config.toml -O /etc/aziot/config.toml
-        sed -i "s#\(connection_string = \).*#\1\"', variables('dcs'), '\"#g" /etc/aziot/config.toml
-
-        iotedge config apply -c /etc/aziot/config.toml
+        if [ ! -z ',variables('dcs'),' ]; then
```

Comment:
```suggestion
        if [ ! -z ', variables('dcs'), ' ]; then
```
I don't know if it matters functionally, but maybe add whitespace just in case and consistency. Otherwise looks good to me, if this works correctly in your tests.

Reply:
fixed with latest commit! and yes i verified that it works.